src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -120,6 +131,33 @@ def test_conversion_as_lone_interface(self):
                 self.run_custom_checks()
 
 
+def _create_mock_probe(*, num_channels: int):
```

Comment:
Also, would you mind moving this to the [`_mock` submodule](https://github.com/catalystneuro/neuroconv/blob/9afd9a9efdf8a5fa4c6f97244b3ec698bdecfdd0/src/neuroconv/tools/testing/_mock/_mock_dataset_models.py) to be nearby other similar helpers?

Reply:
Oh uhh I guess those other ones ought to be in `_mock` too, will fix that in follow-up then

Thanks! This is fine for now